src/codec/encoder/MayaEncoder.cpp

```diff
@@ -86,6 +86,13 @@ struct SerializedGeometry {
 		vertexIndices.reserve(numIndices);
 		normalIndices.reserve(numIndices);
 	}
+
+	bool isEmpty() const {
+		if (coords.empty() || counts.empty() || vertexIndices.empty()) {
```

Comment:
the whole function could be reduced to
```suggestion
		return (coords.empty() || counts.empty() || vertexIndices.empty());
```